SofaKernel/modules/SofaBaseMechanics/BarycentricMapping.inl

```diff
@@ -3727,6 +3727,82 @@ void BarycentricMapping<TIn, TOut>::handleTopologyChange ( core::topology::Topol
     reinit(); // we now recompute the entire mapping when there is a topologychange
 }
 
+#ifdef BARYCENTRIC_MAPPER_TOPOCHANGE_REINIT
```

Comment:
The typdef `BARYCENTRIC_MAPPER_TOPOCHANGE_REINIT` is never defined in this PR. In which case should it be used?